tensorbay/opendataset/HKD/loader.py

```diff
@@ -5,7 +5,6 @@
 # pylint: disable=invalid-name
 
 """Dataloader for Finger Tapping Hand Keypoints Dataset(HKD)."""
```

Comment:
Add the scope in the commit message.
And add the reason why the change is made to the description or the code comments.